c-scape/src/use_libc.rs

```diff
@@ -1,5 +1,23 @@
 //! Utilities to check against C signatures, when enabled.
 
+/// casts the given pointer to another type, similarly to calling `cast()` on it,
+/// while verifying that the layout of the pointee stays the same after the cast
```

Comment:
Here and in the comments for `Pad`, would you mind capitalizing the first letter, and appending a `.`, to sentences and sentence fragments in comments? This is the style used in the rest of the codebase for block comments, and at least to me, it makes them easier to visually parse.